docs/Commands.md

```diff
@@ -4,9 +4,15 @@
 
 ![XML Commands](images/Commands/XMLCommands.png)
 
+## Bind to grammar/schema file
+
+This command triggers the [XML Binding Wizard]() for the current file.

```

Comment:
I think we should document the XML Binding Wizard feature somewhere (unless I missed it) so I can link it here.
Thoughts @angelozerr?

Reply:
If so, it can also be linked in #562

Reply:
Indeed, we should have a new `BindingWithGrammar.md` file referenced in https://github.com/redhat-developer/vscode-xml/tree/master/docs#user-guide.


BindingWithGrammar.md could looks like:

## Binding with  existing grammar

In the following sections, we would like to bind the XML `foo.xml` file:

```xml
<foo>
  <bar />
</foo>
```

with the XSD `foo.xsd` file (in the same folder as foo.xml) :

```xml
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
  <xs:element name="foo">
    <xs:complexType>
      <xs:sequence>
        <xs:element name="bar" />
      </xs:sequence>
    </xs:complexType>
  </xs:element>
</xs:schema>
```

### Command

### Codelens

### Quickfix

## Binding with new grammar

Imagine you have the XML `foo.xml` file:

```xml
<foo>
  <bar />
</foo>
```

and you want to generate a grammar and bind it.

### Quickfix

Reply:
Should I open a separate issue and pr for this new page then?